main.tf

```diff
@@ -29,6 +30,7 @@ resource "google_project_service" "storage_api" {
 resource "google_storage_bucket" "gcs_bucket" {
   name                        = local.bucket_name
   location                    = local.bucket_location
+  storage_class               = local.bucket_storage_class 
```

Comment:
this can be `var.storage_class ` itself. As we are not transforming it to anything else in local.